prerendering.bs

```diff
@@ -658,6 +673,37 @@ The <dfn attribute for="PerformanceNavigationTiming">activationStart</dfn> gette
 
 1. Return a {{DOMHighResTimeStamp}} with a time value equal to the [=current document=]'s [=Document/activation start time=].
 
+<h3 id="client hint cache">Interaction with Client Hint Cache</h3>
+
+Each [=prerendering browsing context=] maintains its <dfn for="prerendering browsing context">prerender scoped Accept-CH cache</dfn>, to persist the origin's opt-in to receive the indicated header fields.
+
+[=prerendering browsing context/Prerender scoped Accept-CH cache=] is an [=ordered map=] of [=origin=] to [=client hints set=].
+
+<div algorithm="add a new prerender Accept-CH cache entry">
+
+Similar to [=add a new Accept-CH cache entry=], to <dfn>add a new prerender Accept-CH cache entry</dfn> to the [=prerendering browsing context/prerender scoped Accept-CH cache=],
+given an |origin| and a [=client hints set=] |hintSet|,
+[=map/set=] [=prerendering browsing context/prerender scoped Accept-CH cache=][|origin|] to |hintSet|.
+
+</div>
+
+<div algorithm="append client hints to request patch">
+  Modify the <a spec=CLIENT-HINTS-INFRASTRUCTURE>append client hints to request</a> algorithm, by prepending the following steps before iterating over the |hintSet|:
+
+  1. Let |browsingContext| be |settingsObject|’s [=global object=]'s [=browsing context=].
+    If |browsingContext| is a [=prerendering browsing context=], retrieve |prerender client hint set| in [=prerendering browsing context/prerender scoped Accept-CH cache=], and then update the |hintSet| with |prerender client hint set|.
```

Comment:
I think you can replace the "retrieve ... and then update" step with something like

```
Set |hintSet| to the [=set/union=] of |hintSet| and |browsingContext|'s [=prerendering browsing context/prerender scoped Accept-CH cache=][|origin|].
```

However you'll need to get `|origin|` from somewhere... maybe from `|request|`?

Reply:
use |request|'s for now.

The setting object also has an orign field. Which one would be better?
https://html.spec.whatwg.org/multipage/webappapis.html#concept-settings-object-origin?